1.1/spec/11-Part-08.adoc

```diff
@@ -797,6 +862,14 @@ geof:minZ (geom: ogc:geomLiteral): xsd:double
 
 Returns the minimum Z coordinate for `geom`.
 
+```
+geof:transform (geom: ogc:geomLiteral, srsURI: xsd:anyURI): ogc:geomLiteral
```

Comment:
`srsURI` needs to be `srsIRI`. I would fix this myself but @situx must not have checked the box to allow commits directly to this branch as I don't seem to have privileges to do it.